### ISSUE TITLE
'Media' background overlay issue on repeatable subform

### ISSUE THREAD

**Issue by ciar4n** [joomla/joomla-cms#11755]
If using a media field type within a repeatable subform there is an issue with the background overlay of the media lightbox once the number of new subforms extends beyond the viewport. The background overlay of the media field lightbox does not extend to the full height of the screen

Subform code example...

```
<?xml version="1.0" encoding="UTF-8"?>
<form>
	<fieldset name="images" class="image-subform">
		<field name="j51_image" type="media" default="" label="MOD_J51_IMAGE"  description="MOD_J51_IMAGE_DESC" size="30" />
		<field name="j51_text" type="textarea" filter="raw" rows="4" cols="42" default="" label="MOD_J51_IMAGETEXT" description="MOD_J51_IMAGETEXT_DESC" />
	</fieldset>
</form>
```

1. Continue to add subforms until the page height is increased
2. Click 'Select' on the media field of the last subform

### Result

![screen shot 2016-08-23 at 05 28 08](https://issues.joomla.org/uploads/1/62b4e8ad4671c3bdad233345bab50233.jpg)

Backgound overlay does not extend to the full height of the screen.

### System information
Joomla 3.6.2

**Comment by ciar4n** [comment]
Possible CSS fix....


```
#sbox-overlay {
    position: fixed;
}

```<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11755">issues.joomla.org/joomla-cms/11755</a>.</sub>

**Comment by C-Lodder** [comment]
Yes, this works. Please submit a PR for it

**Comment by zero-24** [comment]
If you don't know how pull requests works please have a look here: https://docs.joomla.org/Using_the_Github_UI_to_Make_Pull_Requests

**Comment by brianteeman** [comment]
Closed as we have a pr for testing see #11760
